### ISSUE TITLE
OSGI archetype crashes on MAC

### ISSUE THREAD

**Issue by ronsmits** [edvin/tornadofx#183]
- download the felix-tornadofx-5.4.0.zip
- create a maven project using the tornadofx-maven-osgi-project
- cp the jar from the project to the bundle directory
- start felix
- felix starts throwing CSS parser errors following window comes up

when you remove the Styles.class reference in DashboardApp.kt (and clean the felix cache) it will startup correctly.

<img width="770" alt="screenshot 2016-09-16 21 22 34" src="https://cloud.githubusercontent.com/assets/595350/18598648/2fa2ea94-7c54-11e6-9af6-a9dbd063b6c9.png">

**Comment by edvin** [comment]
I think I saw this a while back and fixed it, I will try with 1.5.6-SNAPSHOT and get back to you :)

**Comment by ronsmits** [comment]
Dont know if this is relevant:
````
felix-framework-5.4.0 $ java -jar bin/felix.jar
Waiting for JavaFX Runtime Startup____________________________
Welcome to Apache Felix Gogo

g! .[Done]
Waiting for Primary Stage to be initialized..[Done]
Sep 16, 2016 9:24:41 PM com.sun.javafx.css.parser.CSSParser term
WARNING: CSS Error parsing '*{-fx-background-color: ;}: Unexpected token ';' at [1,24]
````

This is with the standard code from the maven project with the `Styles::class` removed

**Comment by edvin** [comment]
What's the code in the bundle you created?

**Comment by edvin** [comment]
I can confirm that I see the same behavior. Working on it now.

**Comment by edvin** [comment]
This was solved in TornadoFX 1.5.5, but the bundle included 1.5.4. I have upgraded it to include 1.5.6-SNAPSHOT (which will probably be the release or very close to it), so now it should work :)
